extensionsGallery.json

```diff
@@ -3537,6 +3537,63 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "77",
```

Comment:
you should simply copy the kusto extension entry ( you are using a different ID now.) in insider gallery and add it to the same order as insider gallery. makes it easier to compare the differences between insider and stable gallery files

Reply:
Got it, changing 👍